LinksAwakeningClient.py

```diff
@@ -741,8 +742,7 @@ async def deathlink():
                 await asyncio.sleep(1.0)
 
 def run_game(romfile: str) -> None:
-    auto_start = typing.cast(typing.Union[bool, str],
-                            Utils.get_options()["ladx_options"].get("rom_start", True))
+    auto_start = typing.cast(typing.Union[bool, str], LinksAwakeningWorld.settings.get("rom_start", True))

```

Comment:
```suggestion
    auto_start = typing.cast(typing.Union[bool, str], LinksAwakeningWorld.settings.rom_start)

```
Fairly  sure one of the points of the settings API  is to ensure the value exists  and  defaults correctly.